crates/yas_core/src/contracts/yas_pool.cairo

```diff
@@ -23,11 +36,13 @@ trait IYASPool<TContractState> {
     ) -> (u256, u256);
     fn token_0(self: @TContractState) -> ContractAddress;
     fn token_1(self: @TContractState) -> ContractAddress;
+    fn slot_0(self: @TContractState) -> (FixedType, i32, u8, bool);
```

Comment:
This implementation will collide with the one made in PR #135 :
`fn slot_0(self: @TContractState) -> Slot0;`

Reply:
✔️